workers/unity/Assets/Playground/Config/PlayerTemplate.cs

```diff
@@ -32,5 +36,57 @@ public static EntityTemplate CreatePlayerEntityTemplate(string workerId, Improba
 
             return entityBuilder.Build();
         }
+
+        private static void MinimapExample()
+        {
+            var playerConstraint = Interest
+                .Query(Constraint.All(
+                    Constraint.RelativeSphere(20), 
+                    Constraint.Component<PlayerInfo.Component>()))
+                .Result<Position.Component, PlayerInfo.Component>();
+
+            var miniMapConstraint = Interest
+                .Query(Constraint.All(
+                    Constraint.RelativeBox(50, double.PositiveInfinity, 50),
+                    Constraint.Component<MinimapRepresentation.Component>()))
+                .Result<Position.Component, MinimapRepresentation.Component>();
+
+            var interest = new InterestComponent()
+                .AddQuery<PlayerControls.Component>(playerConstraint)
+                .AddQuery<PlayerControls.Component>(miniMapConstraint);
+        }
+        
+        private static void TeamsExample()
+        {
+            var isblue = true;
+            //some logic to determine which team
+
+            var teamQuery = Interest
+                .Query(Constraint.Component(isblue ? BlueTeam.ComponentId : RedTeam.ComponentId));
+            
+            var interest = new InterestComponent()
+                .AddQuery<PlayerControls.Component>(teamQuery);
+        }
+
+        private static void FrequencyExample()
+        {
+            var playerConstraint = Interest
+                .Query(Constraint.All(
+                    Constraint.RelativeSphere(20), 
+                    Constraint.Component<PlayerInfo.Component>()))
+                .Frequency(20)
+                .Result<Position.Component, PlayerInfo.Component>();
+
+            var miniMapConstraint = Interest
+                .Query(Constraint.All(
+                    Constraint.RelativeBox(50, double.PositiveInfinity, 50),
+                    Constraint.Component<MinimapRepresentation.Component>()))
+                .Frequency(1)
+                .Result<Position.Component, MinimapRepresentation.Component>();
+
+            var interest = new InterestComponent()
+                .AddQuery<PlayerControls.Component>(playerConstraint)
+                .AddQuery<PlayerControls.Component>(miniMapConstraint);
+        }
```

Comment:
Can you show how the interest would be added to a template?

Reply:
so once you have have your `var interest = new InterestComponent...`

just add that to your current template e.g.
```
var entityBuilder = ...
   ...
   ...
   .AddComponent(interest, workerAttribute);
```

there's an implicit conversion from `InterestComponent` to `ComponentData` that calls `CreateSchemaComponentData` under the hood